conf/docker.config

```diff
@@ -9,6 +9,13 @@ process {
 //        container = "ghcr.io/torch-consortium/magma/misc:2.0.0-beta"
 //    }
 
+
+    withName:
+    'ISMAPPER.*' {
+        container = "quay.io/biocontainers/ismapper:2.0--py_1"
+    }
+
```

Comment:
Once the branch is finalized, I'll add this dependency into the existing `magma-container` 

CC @Mxrcon